.laminas-ci.json

```diff
@@ -1,7 +1,8 @@
 {
     "ignore_php_platform_requirements": {
         "8.2": false,
-        "8.3": false
+        "8.3": false,
+        "8.4": true
```

Comment:
this should can list 8.4 only, remove 8.2 and 8.3 from the list

Reply:
Done